docs/docs/examples-tutorials/recipes/running-tracetest-with-dynatrace.md

```diff
@@ -0,0 +1,356 @@
+# Running Tracetest With Datadog
+
+:::note
+[Check out the source code on GitHub here.](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-datadog)
+:::
+
+[Tracetest](https://tracetest.io/) is a testing tool based on [OpenTelemetry](https://opentelemetry.io/) that allows you to test your distributed application. It allows you to use data from distributed traces generated by OpenTelemetry to validate and assert if your application has the desired behavior defined by your test definitions.
+
+[Datadog](https://www.datadoghq.com/) is an observability solution for cloud-scale applications, providing solutions to monitor databases, servers, tools and services. It provides integrated distributed tracing, logs and metrics solutions and supports OpenTelemetry standards.
+
+## OpenTelemetry Demo `v1.3.0` with Datadog, OpenTelemetry and Tracetest
+
+This is a simple sample app on how to configure the [OpenTelemetry Demo `v1.3.0`](https://github.com/open-telemetry/opentelemetry-demo) to use [Tracetest](https://tracetest.io/) for enhancing your E2E and integration tests with trace-based testing and [Datadog](https://www.datadoghq.com/) as a trace data store.
+
+## Prerequisites
+
+You will need [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your machine to run this sample app! Additionally, you will need a Datadog account and an API. Sign up to Datadog on their [homepage](https://www.datadoghq.com/) by clicking on `Get Started Free`.
```

Comment:
```suggestion
You will need [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your machine to run this sample app! Additionally, you will need a Datadog account and an API Key. Sign up for Dynatrace on their [homepage](https://www.dynatrace.com/) by clicking on `Free trial`.
```

Reply:
@danielbdias is going to review this today. If we can get it knocked out, we may be able to slide it into the RC on Monday and then release on Tuesday!